Makefile

```
@@ -92,6 +95,11 @@ discoverycache:
 	KUBECONFIG=admin.kubeconfig go run ./hack/gendiscoverycache
 	$(MAKE) generate
 
+fix-macos-vendor:
+ifeq ($(shell uname -s),Darwin)
+	mv ./vendor/github.com/Microsoft ./vendor/github.com/microsoft || true
```

```suggestion
	mv ./vendor/github.com/Microsoft ./vendor/github.com/temp-microsoft && mv ./vendor/github.com/temp-microsoft ./vendor/github.com/microsoft || true
```

Suggesting this change because it doesn't work on Mac as-is. If we try to `mv` the directory without an intermediate rename, `mv` thinks we want to put the directory inside of itself.